homeassistant/components/frontend/templates/index.html

```diff
@@ -8,6 +8,8 @@
     <link rel='icon' href='/static/icons/favicon.ico'>
     <link rel='apple-touch-icon' sizes='180x180'
           href='/static/icons/favicon-apple-180x180.png'>
+    <link rel="mask-icon" href="/static/icons/favicon-apple-180x180.png" color="#3fbbf4">
```

Comment:
Looks like the icon needs to be an svg icon.

We have one here but the SVG should be simplified
https://github.com/home-assistant/home-assistant-assets/blob/master/logo_v2.svg

https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariWebContent/pinnedTabs/pinnedTabs.html